aadl.library/AADL_example_0001.sysml

```diff
@@ -0,0 +1,63 @@
+/* SysML v2 Domain Library for AADL:

```

Comment:
This example is not a part of the library a should be excluded from this Pull Request. It might make sense to create a separate directory or even a separate repo for examples. Again, we should probably discuss where we can put examples.

Reply:
But we do need a way to store an exchange examples that are associated with a precise version of the library
Either we agree on some version numbering or we put library and examples in the same commits
Up to you.